index.d.ts

```diff
@@ -1,9 +1,6 @@
 /* Any copyright is dedicated to the Public Domain.
  * http://creativecommons.org/publicdomain/zero/1.0/ */
-/// <reference path="./types.d.ts"/>
+
 import { CompatData } from './types';
 
-// This is necessary to have intellisense in projects which
-// import data from this package.
-declare const compatData: CompatData;
-export = compatData;
```

Comment:
Does replacing this with `export default CompatData` fix the Intellisense issues?  (I don't really use IntelliJ so I am unsure!)